### PR TITLE
Shooter

### PR DESCRIPTION
**Pull request by SaltyJoke**

### CODE REVIEW

File: src/main/java/frc/robot/commands/EjectCell.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.Shooter;
+
+public class EjectCell extends CommandBase {
```
Comment (alexander-mcdowell) [inline src/main/java/frc/robot/commands/EjectCell.java]: What is the difference between the EjectCell command and the Shoot command? It is not entirely clear.

File: src/main/java/org/team199/robot2020/Robot.java [hunk 2]
```diff
@@ -20,12 +24,16 @@
 public class Robot extends TimedRobot {
   private RobotContainer robotContainer;
 
+  private static Shooter shooter;
```
Comment (kevinzwang) [inline src/main/java/org/team199/robot2020/Robot.java]: this subsystem should be in RobotContainer just like how Drivetrain is

File: src/main/java/org/team199/robot2020/commands/Shoot.java [hunk 3]
```diff
@@ -0,0 +1,26 @@
+package org.team199.robot2020.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import edu.wpi.first.wpilibj2.command.CommandScheduler;
+import org.team199.robot2020.subsystems.Shooter;
+import org.team199.robot2020.commands.EjectCell;
+
+public class Shoot extends CommandBase {
```
Comment (kevinzwang) [inline src/main/java/org/team199/robot2020/commands/Shoot.java:7]: the two commands `Shoot` and `EjectCell` are confusingly named. At least add comments to their constructors!

File: src/main/java/org/team199/robot2020/commands/ShooterTargetSpeed.java [hunk 4]
```diff
@@ -0,0 +1,23 @@
+package org.team199.robot2020.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import org.team199.robot2020.subsystems.Shooter;
+
+public class ShooterTargetSpeed extends CommandBase {
```
Comment (kevinzwang) [inline src/main/java/org/team199/robot2020/commands/ShooterTargetSpeed.java]: Why does the `Shoot` command seem to just be doing what this command should do?

File: src/main/java/org/team199/robot2020/subsystems/Shooter.java [hunk 5]
```diff
@@ -0,0 +1,25 @@
+package org.team199.robot2020.subsystems;
+
+import edu.wpi.first.wpilibj2.command.Subsystem;
+
+public class Shooter implements Subsystem {
```
Comment (kevinzwang) [inline src/main/java/org/team199/robot2020/subsystems/Shooter.java]: Use `extends SubsystemBase` not `implements Subsystem`.

**Review by kevinzwang — CHANGES_REQUESTED** [review]

**Pull request merged**